README.md

```diff
@@ -67,15 +67,13 @@ To install the resulting binary, run
 cargo install --path . --locked
 ```
 
-### Installing with Cargo
```

Comment:
I'd rather theep the header - it breaks the explanation up into sections because they're two separate methods.

```md
### Alternative: Compiling without sources
```

While not strictly true, I think this is the functional difference between `build` and `install`: The user doesn't need to prepare the source directory.

Reply:
thats fair, i do think it should be a subsection of the part that talks about dependencies, so that would be 4 `#`